lib/rubocop/cop/style/format_string.rb

```diff
@@ -40,6 +40,50 @@ def message(detected_style)
         def method_name(style_name)
           style_name == :percent ? 'String#%' : style_name
         end
+
+        def autocorrect(node)
+          lambda do |corrector|
+            _, method = *node
+            if method == :%
```

Comment:
I tried to refactor. how about this?  https://github.com/bbatsov/rubocop/pull/4354/commits/69160fc737d26cde60bfd41be1ff389d2adbf647